spec/cli_v1_commands.yaml

```diff
@@ -206,7 +206,7 @@ commands:
           SERVICE_ROLE_KEY="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6InNlcnZpY2Vfcm9sZSIsImV4cCI6MTk4MzgxMjk5Nn0.EGIM96RAZx35lJzdJsyH-qQwv8Hdp7fsn3W0YpN81IU"
           STUDIO_URL="http://localhost:54323"
       - id: output-custom-name
-        name: Customize the names of exported variables
```

Comment:
not sure about this change - I thought we follow US spelling.

Reply:
Yep, should be American English spelling.

Reply:
Actually I think this was generated from outdated cli repo. Could you rebase and regenerate? https://github.com/supabase/cli/blob/main/docs/templates/examples.yaml#L91

Reply:
nvm, I've updated cli.